config/cron.hjson

```diff
@@ -165,7 +165,7 @@
             )
             select 
                 f.id, 
-                f.assignment_id, 
+                cast(f.assignment_id as bigint),
```

Comment:
Was there any reason this change was made?

Reply:
This is to have the same type as the assignment_id field from the udp query. 

Plus the assignment table inside MyLA database already have id field as bigint(20) type.